core-services/dashboard-analytics/src/main/java/com/tarento/analytics/helper/SortingHelper.java

```diff
@@ -0,0 +1,50 @@
+package com.tarento.analytics.helper;
+import org.springframework.stereotype.Component;
+import com.tarento.analytics.dto.Data;
+import com.tarento.analytics.dto.Plot;
+import java.util.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import static com.tarento.analytics.handler.IResponseHandler.*;
+
+@Component
+public class SortingHelper {
+    public List<Data> sort(String sortingKey, List<Data> dataList) {
+        Comparator<Plot> plotSortComparator = PlotSortComparator(sortingKey, dataList);
```

Comment:
@bhanuprakash-egov 

- We need not pass the entire `dataList` as the argument. A boolean can be arrived based on the size and the boolean value can be passed to the comparator

```java
Boolean isValueSortApplicable = dataList.size() == 1;
        Comparator<Plot> plotComparator = getPlotComparator(sortingKey, isValueSortApplicable);
```

- Use camelCase for the private function name